docker/install_libs.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+
+LIBNAME="${1:-support}"
+LIVERSION="${2:-0.3.0}"
+BASEURL='https://github.com/bats-core'
+DESTDIR="${BATS_LIBS_DEST_DIR:-/usr/lib}"
+TMPDIR=$(mktemp -d -t bats-libs-XXXXXX)
+USAGE="Please provide the bats libe name and version \nFor example: install_libs.sh support 2.0.0\n"
+
+trap "test -d "${TMPDIR}" && rm -fr "${TMPDIR}"" EXIT
+
+_log() {
+    printf "$(date "+%Y-%m-%d %H:%M:%S") - %s - %s\n" "${1}" "${2}"
+}
+
+create_temp_dirs() {
+    mkdir -p "${TMPDIR}/${1}"
+    [[ ${LIBNAME} != "detik" ]] && mkdir -p "${DESTDIR}/bats-${1}/src" || _log INFO "Skipping src 'cause Detik does not need it"
```

Comment:
I think an `if else` would be more readable (and less subtle in its semantics)

Reply:
done